Pull Request Raiymbekov Daniyar

DaniyarRa opened the pull request.

DaniyarRa committed:
Practice Task N1

DaniyarRa committed:
Practice Task N1

DaniyarRa committed:
Practice Task N1

DaniyarRa committed:
Merge remote-tracking branch 'origin/main'

DaniyarRa committed:
Practice 2 Task

DaniyarRa committed:
Practice 2 Task

DaniyarRa committed:
Practice 3 Task

DaniyarRa committed:
Practice 3 Task

arscool3 commented:
nice, but add other practices please

DaniyarRa committed:
Group Task: Team Members: Mussabekov Madi, Raiymbekov Daniyar

DaniyarRa committed:
Group Task: Team Members: Mussabekov Madi, Raiymbekov Daniyar

DaniyarRa committed:
Update Forked Repo

DaniyarRa committed:
Midterm Part1

DaniyarRa committed:
Midterm Part1

DaniyarRa committed:
Midterm Part2

DaniyarRa committed:
Midterm Part2

DaniyarRa committed:
Midterm Part3

DaniyarRa committed:
Midterm Part3

DaniyarRa committed:
Midterm Part4

DaniyarRa committed:
Midterm Part4

DaniyarRa committed:
Merge branch 'arscool3:main' into main

DaniyarRa committed:
Merge remote-tracking branch 'origin/main'

DaniyarRa committed:
Midterm Final

DaniyarRa committed:
Midterm Final

DaniyarRa committed:
Midterm Final

DaniyarRa committed:
Midterm Final

arscool3 commented on Midterm/main/templates/entry_page.html:
```diff
@@ -0,0 +1,57 @@
+<!DOCTYPE html>
```
можете пожалуйста добавить template filters

DaniyarRa committed:
Midterm Final

DaniyarRa committed:
Midterm Final

arscool3 commented:
гуд, 25 / 25

DaniyarRa committed:
Practice #6

DaniyarRa committed:
Merge remote-tracking branch 'origin/main'

# Conflicts:
#	Midterm/main/templates/flight_option.html
#	Midterm/main/urls.py
#	Midterm/main/views.py
#	Midterm/midterm/settings.py

DaniyarRa committed:
Merge branch 'arscool3:main' into main

DaniyarRa committed:
Practice #7

DaniyarRa committed:
Merge remote-tracking branch 'origin/main'

DaniyarRa committed:
Practice #8

DaniyarRa committed:
Arranging practices 1,2,3.
New practices 6,7(they integrated in logic of Midterm, so that their content exact same).
Partly practice 4.

DaniyarRa committed:
Merge branch 'arscool3:main' into main

DaniyarRa committed:
Merge remote-tracking branch 'origin/main'

DaniyarRa committed:
Arranging practices 1,2,3.
New practices 6,7(they integrated in logic of Midterm, so that their content exact same).
Partly practice 4.

DaniyarRa committed:
Practice 4.

DaniyarRa committed:
Practice 9

DaniyarRa committed:
Practice 10

DaniyarRa committed:
Practice 11

DaniyarRa committed:
#1. Final Project Structure

arscool3 commented on Practice11/main.py:
```diff
@@ -0,0 +1,58 @@
+app = FastAPI()
+
+# Define a basic HTML page for WebSocket connection
+html_page = """
```
крутая имплементация

arscool3 commented:
nice 60 / 60

DaniyarRa committed:
#2. User Table. Connection with DB

DaniyarRa committed:
#3. Tokens, Auth Router, new Models

DaniyarRa committed:
#3. Tokens, Auth Router, new Models

DaniyarRa committed:
#4. Gmail Application Confirmation

DaniyarRa committed:
#4. Gmail Application Confirmation

DaniyarRa committed:
#4. Gmail Application Confirmation

DaniyarRa committed:
#5. Hot Fixes Application.status_id

DaniyarRa committed:
#6. Hot Fixes Application.status_id #2

DaniyarRa committed:
#6. Logout fixes
